app/lib/primer/classify/cache.rb

```diff
@@ -19,26 +19,6 @@ def clear!
         end
 
         def preload!
-          preload(
-            keys: Primer::Classify::Spacing::MARGIN_DIRECTION_MAPPINGS.keys,
-            values: Primer::Classify::Spacing::MARGIN_DIRECTION_OPTIONS
-          )
-
-          preload(
-            keys: Primer::Classify::Spacing::BASE_MAPPINGS.keys,
-            values: Primer::Classify::Spacing::BASE_OPTIONS
-          )
-
-          preload(
-            keys: Primer::Classify::Spacing::AUTO_MAPPINGS.keys,
-            values: Primer::Classify::Spacing::AUTO_OPTIONS
-          )
-
-          preload(
-            keys: Primer::Classify::Spacing::RESPONSIVE_MAPPINGS.keys,
-            values: Primer::Classify::Spacing::RESPONSIVE_OPTIONS
-          )
-
```

Comment:
I could add these back, but spent a little time trying and found that they didn't improve the bench tests.

Reply:
🤔 it's curious to me that this didn't change the benchmark. I guess that means your new solution is just as fast as preloading? 🤯

Reply:
I don't think we'll need to preload anything generated from YAML files since it should be implicitly cached, and won't impact our object allocation tests.